researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/BridgeDataProvider.java

```diff
@@ -484,11 +489,16 @@ public void setUserSharingScope(Context context, String scope) {
 
     @Override
     public SchedulesAndTasksModel loadTasksAndSchedules(Context context) {
-        // TODO: integrate with bridge
-        // forConsentedUsersApi.getSchedules();
-        // forConsentedUsersApi.getScheduledActivities();
+        logger.info("loadTasksAndSchedules()");
+
+        // TODO: figure out the correct arguments to pass here
+        ScheduledActivityList scheduledActivityList = bridgeManagerProvider.getActivityManager().getActivities("-07:00", 1, 0)
```

Comment:
Totally agree.  I have already made that change locally and will include in the next update.